.github/workflows/build-webhook-configuration-container.yml

```diff
@@ -0,0 +1,104 @@
+name: Build Webhook Configuration
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - .github/actions/build-component-per-arch/**
+      - .github/actions/build-component-multi-arch/**
+      - .github/workflows/build-webhook-configuration-container.yml
+      - build/containers/Dockerfile.webhook-configuration
+      - webhooks/validating/configuration
+      - version.txt
+      - build/akri-containers.mk
+      - Makefile
+  pull_request:
+    branches: [main]
+    paths:
+      - .github/actions/build-component-per-arch/**
+      - .github/actions/build-component-multi-arch/**
+      - .github/workflows/build-workflow-configuration-container.yml
+      - build/containers/Dockerfile.workflow-configuration
+      - webhooks/validating/configuration
+      - version.txt
+      - build/akri-containers.mk
+      - Makefile
+  release:
+    types:
+      - published
+
+env:
+  AKRI_COMPONENT: webhook-configuration
+  MAKEFILE_COMPONENT: webhook-configuration
+
+jobs:
+  per-arch:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    strategy:
+      matrix:
+        arch:
+          - arm64v8
+          - arm32v7
+          - amd64
+
+    steps:
+      - name: Checkout the head commit of the branch
+        uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+
+      - name: Prepare To Install
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Install Deps
+        run: |
+          yarn install
+          yarn add @actions/core @actions/github @actions/exec fs
+
+      - name: Run Per-Arch component build for ${{ env.AKRI_COMPONENT }}
+        uses: ./.github/actions/build-component-per-arch
+        with:
+          github_event_name: ${{ github.event_name }}
+          github_ref: ${{ github.ref }}
+          github_event_action: ${{ github.event.action }}
+          github_merged: ${{ github.event.pull_request.merged }}
+          container_name: ${{ env.AKRI_COMPONENT }}
+          container_prefix: ghcr.io/deislabs/akri
+          container_registry_base_url: ghcr.io
+          container_registry_username: ${{ secrets.crUsername }}
+          container_registry_password: ${{ secrets.crPassword }}
+          makefile_component_name: ${{ env.MAKEFILE_COMPONENT }}
+          platform: ${{ matrix.arch }}
+          build_rust: "0"
```

Comment:
if you specify `build_rust: "1"` here, it should take care of building your component for you.

Reply:
Thank you! (https://github.com/deislabs/akri/pull/206/commits/106c27ddb60e316e52a6fd75ac1aff5e2ce72596)